src/SprykerEco/Yves/Unzer/Theme/default/views/sofort/sofort.twig

```diff
@@ -0,0 +1,9 @@
+{% extends model('template') %}
+
+{% define data = {
+    form: form
+} %}
+
```

Comment:
Please add a template for all of these views to avoid duplication.

Reply:
As discussed, not changed.